opennms-base-assembly/src/main/filtered/etc/telemetryd-configuration.xml

```diff
@@ -75,13 +77,28 @@
                 </rrd>
             </package>
         </adapter>
+
+        <adapter name="Netflow-5-Telemetry" class-name="org.opennms.netmgt.telemetry.protocols.netflow.adapter.netflow5.Netflow5TelemetryAdapter" enabled="false">
+            <parameter key="script" value="${install.dir}/etc/telemetryd-adapters/netflow-telemetry.groovy"/>
+            <package name="Netflow-5-Default">
+                <rrd step="300">
+                    <rra>RRA:AVERAGE:0.5:1:2016</rra>
+                    <rra>RRA:AVERAGE:0.5:12:1488</rra>
+                    <rra>RRA:AVERAGE:0.5:288:366</rra>
+                    <rra>RRA:MAX:0.5:288:366</rra>
+                    <rra>RRA:MIN:0.5:288:366</rra>
+                </rrd>
+            </package>
+        </adapter>
     </queue>
 
     <!-- Netflow v9 listener & adapters -->
     <listener name="Netflow-9-UDP-4729" class-name="org.opennms.netmgt.telemetry.listeners.UdpListener" enabled="false">
         <parameter key="port" value="4729"/>
 
-        <parser name="Netflow-9-Parser" class-name="org.opennms.netmgt.telemetry.protocols.netflow.parser.Netflow9UdpParser" queue="Netflow-9" />
+        <parser name="Netflow-9-Parser" class-name="org.opennms.netmgt.telemetry.protocols.netflow.parser.Netflow9UdpParser" queue="Netflow-9">
+            <parameter key="includeRawMessage" value="false"/>
```

Comment:
Do we really need to add this parameter in the default config for all parsers ?  may be just need to be specified in docs and keep the default as false.

Reply:
done.